package.json

```diff
@@ -56,9 +56,9 @@
     "foreach": "2.0.5",
     "handlebars": "2.0.0",
     "insert-css": "0.2.0",
-    "leaflet": "^0.7.5",
-    "leaflet-plugins": "^1.3.10",
-    "leaflet.markercluster": "^0.4.0",
+    "leaflet": "1.0.3",
+    "leaflet.gridlayer.googlemutant": "0.4.3",
```

Comment:
The only thing we were using from leaflet-plugins was the Google tile layer plugin. That's non-existent in the latest leaflet-plugins, googlemutant is the new black apparently